src/library/f3d_loader.h

```diff
@@ -75,27 +73,40 @@ class F3DLoader
    * filePath the path to the file and fileInfo a more complete information about the file
    * Return true if there is a file or false otherwise
    */
-  void GetNextFile(
+  void getFileInfo(
     LoadFileEnum load, int& nextFileIndex, std::string& filePath, std::string& fileInfo) const;
 
   /**
    * Start the interaction/rendering
    * Return true if rendering and interaction is successful, false otherwise
+   * TODO Move that to another class ?
    */
-  bool Start();
+  bool start();
 
-  F3DLoader();
-  ~F3DLoader();
+  /**
+   * Forward to AnimationManager to start the animation
+   * TODO Move that to another class ?
+   */
+  void toggleAnimation();
+
+  /**
+   * Initialize the rendering stack managed by the loader
+   * This will initialize the following members:
+   * AnimationManager, RenWin, Interactor, Style
+   * TODO Manage that internally depending on the type of context provided
+   */
+  void InitializeRendering(std::string, bool offscreen, const void* iconBuffer, size_t inconLength);
 
-protected:
-  void InitializeRenderer();
+  /**
+   * Convenience method to get the interactor style
+   * TODO Remove
+   */
+  vtkF3DInteractorStyle* GetInteractorStyle();
```

Comment:
@Meakk : As of now, I do not see how we can remove this method while keeping full fonctionnality of F3D. Please advise

Reply:
It's only used in `F3DStarter` to setup the observers, right?  
Then create a simple `f3d::interactor` holding this `vtkF3DInteractorStyle`. Move `f3d::interactor` to `F3DStarter` and call a `f3d::loader::setInterator`.

Reply:
F3DStarter is supported to go in the application, which is not supposed to link with VTK